components/brave_wallet_ui/common/hardware/types.ts

```diff
@@ -31,11 +31,7 @@ export type HardwareOperationResult = {
 }
 
 export type SignHardwareTransactionOperationResult = HardwareOperationResult & {
```

Comment:
It would be clearer if we change this to be like `SignHardwareOperationResult` which is the result for signing transaction or message